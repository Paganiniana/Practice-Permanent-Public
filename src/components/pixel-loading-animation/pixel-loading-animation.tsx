import React from 'react'
import './pixel-loading-animation.css'

export class PixelLoader extends React.Component {

    render() {
        return <div id="loader"></div>
    }
}
