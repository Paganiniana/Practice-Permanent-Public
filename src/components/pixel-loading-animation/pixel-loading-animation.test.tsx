import React from 'react'
import { render } from '@testing-library/react'
import { PixelLoader } from './pixel-loading-animation'

describe('Pixel Loading Component', () => {
    test('It should render', () => {
        const res = render(<PixelLoader />)
        expect(res.asFragment()).toMatchSnapshot()
    })
})