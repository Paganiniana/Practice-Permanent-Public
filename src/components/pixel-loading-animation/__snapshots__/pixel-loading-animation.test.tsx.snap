// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pixel Loading Component It should render 1`] = `
<DocumentFragment>
  <div
    id="loader"
  />
</DocumentFragment>
`;
